### PR TITLE
Translation + auto linter fixes from prev commits

### PR DESCRIPTION
**Pull request by Darkmift**
This PR focuses on `support.md` translation to hebrew with additional translations in related files involved in rendering said page (surrounding layout,nav etc)

As per guidlines running `yarn check-all` also made some minor linting fixes in some other unrelated files so figured I will not discard these under the assumtion said changes should have been done before and may have been overlooked.

### CODE REVIEW

File: src/components/SocialBanner/SocialBanner.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ export default function SocialBanner() {
           <span
             css={{
               marginRight: '0.5rem',
+              marginLeft: '0.5rem',
             }}>
             {bannerText}
           </span>
```
Comment (auchynnikau) [inline src/components/SocialBanner/SocialBanner.js:58]:
```suggestion
          <span css={{margin: '0 0.5rem'}}>
            {bannerText}
          </span>
```

File: content/community/support.md [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
 permalink: community/support.html
 redirect_from:
   - "support.html"
 ---
 
-React has a community of millions of developers.
+קהילת הפיתוח של ריאקט מורכבת ממיליוני מפתחות ומפתחים.
 
-On this page we've listed some React-related communities that you can be a part of; see the other pages in this section for additional online and in-person learning materials.
+בדף זה ריכזנו קהילות בנושא ריאקט שניתן לקחת חלק בהן,ניתן גם לצפות בדפים נוספים למטה על מנת לקבל חומרים נוספים אונליין ולימוד עצמי.
```
Comment (galnir) [inline content/community/support.md:13]: Maybe
בדף זה ריכזנו קהילות בנושא ריאקט שניתן לקחת חלק בהן, ניתן גם לצפות בדפים נוספים למטה על מנת לקבל חומרי לימוד עצמי.

חומרים נוספים אונליין ולימוד sounds not right

Comment (Darkmift) [reply content/community/support.md:13]: It gets the poit across but I see your point.

File: src/components/SocialBanner/SocialBanner.js (hunk 1 above)
Comment (galnir) [reply src/components/SocialBanner/SocialBanner.js:58]: not necessary

**Pull request merged**